City-of-Pori/pori-kada
# PORI-403 Created small liftup banner block for sections and added it …

…to section front page

To test:

1. drush fra -y && drush cc all
2. Go to add a liftup and flag it to small liftup banner
3. Go to front page and make sure it didn't appear yet.
4. Now go edit it and add "Promote to front page" option.
5. Go to front page and make sure the liftup banner appears.
6. Go to edit it once again and remove the "Promote to front page" checkbox and add the banner to a section.
7. Go to the section front page and make sure the liftup banner appears. Go to the site front page and make sure the liftup banner doesn't appear there.
